tests/python/unittest/test_gluon.py

```diff
@@ -503,6 +503,40 @@ def test_deconv():
     # layer = nn.Conv3DTranspose(16, (3, 3, 3), layout='NDHWC', in_channels=4)
     # # check_layer_forward(layer, (1, 10, 10, 10, 4))
 
+@with_seed()
+def test_deconv_dilation():
```

Comment:
Since deconv is a really important OP,  I suggest to visit the original deconv test cases and add dilation > 1 cases alongside the old tests. This ensures better coverage than this single test case. 
Feel free to keep this unittest which LGTM as well.